### ISSUE TITLE
yarn web-dev runs into issues

### ISSUE THREAD

**Issue by sappelhoff** [bids-standard/legacy-validator#1188]
I was trying to develop the web validator through trial and error and wanted to serve the build locally using `yarn web-dev`, as described in our README.

However, I run into this error (ubuntu 18.04, npm 7.5.4, nodejs10.23.3)

```
yarn run v1.22.5
$ cd bids-validator-web && yarn dev
$ next
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
ready - started server on http://localhost:3000
warn  - Fast Refresh is disabled in your application due to an outdated `react` version. Please upgrade 16.10 or newer! Read more: https://err.sh/next.js/react-version
Warning: Built-in CSS support is being disabled due to custom CSS configuration being detected.
See here for more info: https://err.sh/next.js/built-in-css-disabled

event - compiled successfully
event - build page: /
wait  - compiling...
error - ./node_modules/react-bootstrap/esm/OverlayTrigger.js
Attempted import error: 'useUncontrolledProp' is not exported from 'uncontrollable'.
```

**Comment by sappelhoff** [comment]
as part of PR targeting this, one "save cache" step should be remove from the circleci job:

https://github.com/bids-standard/bids-validator/blob/cd7788fbe97ad8807e761c7aa40a92a45eb931ad/.circleci/config.yml#L39-L42

so that instead, only the following save step will be performed:

https://github.com/bids-standard/bids-validator/blob/cd7788fbe97ad8807e761c7aa40a92a45eb931ad/.circleci/config.yml#L94-L98

with 2 save steps, the "next" cache will always get overridden before it can be used later.

**Comment by sappelhoff** [comment]
this was fixed in #1187 `npm run web-dev` does the job now. I'll update the README, which currently says to run `npm web-dev`, missing the `run`.
